holoviews/plotting/mpl/tabular.py

```diff
@@ -89,7 +89,7 @@ def pprint_value(self, value):
         elif isinstance(value, str):
             formatted = safe_unicode(value)
         else:
-            formatted = str(value)
```

Comment:
``safe_unicode`` will not cast a non-string type to unicode, so if something downstream of this is expecting a string this won't do it.